packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -232,6 +253,35 @@ export class PluginLedgerConnectorCorda
       endpoints.push(endpoint);
     }
 
+    {
+      const opts: IListCPIEndpointV1Options = {
+        apiUrl: this.options.apiUrl,
+        logLevel: this.options.logLevel,
+      };
+      const endpoint = new ListCPIEndpointV1(opts);
+      endpoints.push(endpoint);
+    }
+
+    {
+      const opts: IFlowStatusEndpointV1Options = {
+        apiUrl: this.options.apiUrl,
+        logLevel: this.options.logLevel,
+        holdingIDShortHash: this.options.holdingIDShortHash,
+      };
+      const endpoint = new FlowStatusEndpointV1(opts);
+      endpoints.push(endpoint);
+    }
+
+    {
+      const opts: IFlowStatusResponseEndpointV1Options = {
+        apiUrl: this.options.apiUrl,
+        logLevel: this.options.logLevel,
+        holdingIDShortHash: this.options.holdingIDShortHash,
+        clientRequestID: this.options.clientRequestID,
+      };
+      const endpoint = new FlowStatusResponseEndpointV1(opts);
+      endpoints.push(endpoint);
+    }
     this.log.info(`Instantiated endpoints of ${pkgName}`);
     return endpoints;
   }
```

Comment:
Here we should have the function to call all the flow rpc calls GET/POST.. (instead of having another server client layer for the same)